testing/scenario_app/lib/src/platform_view.dart

```diff
@@ -1533,6 +1531,68 @@ class TwoPlatformViewsWithOtherBackDropFilter extends Scenario
   }
 }
 
+/// A simple platform view for testing backDropFilter with a platform view in the scene.
+///
+/// The backdrop filter sigma value is negative, which tries to reproduce a crash, see:
```

Comment:
curious what does a negative sigma mean? is it not supposed to happen? maybe make sense to print a warning if so.

Reply:
I'm not entirely sure why the negative sigma is used in one of the example code to reproduce the bug, however, the fix is more about accounting the possibility where the filter object is null. 

As a follow up, I think we also need to disable partial repaint when there is a platform view.